input/pagecontent/iti-mhd.md

```diff
@@ -28,8 +31,48 @@ transactions between them.
 The Find Document Lists [[ITI-66]](https://profiles.ihe.net/ITI/MHD/ITI-66.html) transaction defined in [MHD](https://profiles.ihe.net/ITI/MHD/index.html) SHALL not be made available in this context.
 
 ### Actor options  
-For all actors the Comprehensive Metadata Option SHALL be supported. For all actors the Metadata as defined in Annex 3 SHALL be supported.
 
+Options that can be selected for each actor in this profile,are listed in the table below. 
+
+{:class="table table-bordered"}
+| Actor                                         | Option Name         | Optionality  |
+|-----------------------------------------------|---------------------------|-------------|
+| Document Source                               | Comprehensive Metadata <br />Federated Cross Community Access   | R  <br /> see actor option notes    |
```

Comment:
Should a note about Document Source not supporting FXCA in Provide Document Bundle be added?

Reply:
I don't think this is necessary, what would put into the note?